embedchain/vectordb/zilliz.py

```diff
@@ -162,13 +167,16 @@ def query(self, input_query: List[str], n_results: int, where: Dict[str, any], s
                 collection_name=self.config.collection_name,
                 data=[query_vector],
                 limit=n_results,
-                output_fields=["text"],
+                output_fields=output_fields,
             )
 
         doc_list = []
         for query in query_result:
-            doc_list.append(query[0]["entity"]["text"])
-
+            data = query[0]["entity"]
+            content = data["text"]
```

Comment:
Can you please change the variable name from `content` to `context` for all the vector dbs for better understanding?

Reply:
sure!